### PR TITLE
New package: libpurple-facebook-20160125.92885e0456ed

### PR DESCRIPTION
**Pull request by jprjr**
I know that version number seems pretty weird - but that seems to be how the author does releases: https://github.com/jgeboski/purple-facebook/releases

### CODE REVIEW

**Comment by jprjr** [comment]
Oops, got the license wrong. Correcting (it's GPL-2, not 3)

**Comment by jprjr** [comment]
Corrected + pushed.

File: srcpkgs/libpurple-facebook/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'libpurple-facebook'
+
+pkgname="libpurple-facebook"
+version="92885e0456ed"
```
Comment (xtraeme) [inline srcpkgs/libpurple-facebook/template]: This version is weird, how do you know the next one will be greater than the current one?

Comment (jprjr) [inline srcpkgs/libpurple-facebook/template]: I'm not sure. That's how the author versions his releases. Should I maybe prefix the version with some number? 

On February 6, 2016 12:29:41 AM CST, Juan RP <notifications@github.com> wrote:
>> @@ -0,0 +1,15 @@
>> +# Template file for 'libpurple-facebook'
>> +
>> +pkgname="libpurple-facebook"
>> +version="92885e0456ed"
>
>This version is weird, how do you know the next one will be greater
>than the current one?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/voidlinux/void-packages/pull/3544/files#r52098733

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

Comment (jprjr) [inline srcpkgs/libpurple-facebook/template]: Or maybe I could use the commit date as the package version 

On February 6, 2016 12:29:41 AM CST, Juan RP <notifications@github.com> wrote:
>> @@ -0,0 +1,15 @@
>> +# Template file for 'libpurple-facebook'
>> +
>> +pkgname="libpurple-facebook"
>> +version="92885e0456ed"
>
>This version is weird, how do you know the next one will be greater
>than the current one?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/voidlinux/void-packages/pull/3544/files#r52098733

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by jprjr** [comment]
Changed the version to be "$releasedate-$packageversion", like "20160125-92885e0456ed"

**Comment by xtraeme** [comment]
Note that versión must not contain dashes. Try using a dot instead.

Thanks

**Comment by jprjr** [comment]
Sure thing. I got lazy and forgot to replace "${version}" in the distfiles url. This should be good-to-go.

**Pull request merged**